composer.json

```diff
@@ -34,7 +35,7 @@
         "sonata-project/user-bundle": "For user timezone detection"
     },
     "conflict": {
-        "sonata-project/user-bundle": "<2.0 || >=4.0"
+        "sonata-project/user-bundle": "<2.0 || >=5.0"
```

Comment:
Why should we limit ourself without at this stage?

👎 for a upper version limit